fix(bazel): unknown replay compiler error in windows

alan-agius4 opened the pull request:
In Windows the compiler path includes `.exe` thus it will never match and throw  `Unknown replay compiler` in windows.

//cc @alexeagle

alfaproject commented:
You should probably leave a comment explaining why you are checking that the executable `startsWith` instead of being equal to. No one is going to remember this and just think wtf

alan-agius4 committed:
fix(bazel): unknown replay compiler error in windows

In Windows the compiler path ends with `.exe` thus it will never match and throw  `Unknown replay compiler`
